aptos-move/framework/aptos-framework/sources/simple_delegations.move

```diff
@@ -0,0 +1,848 @@
+/// Module that offers convenient simple delegations where a single owner account can delegate
+/// to multiple operators. This does not offer general purpose delegations where funds from multiple
```

Comment:
this doesn't compute. it is a contract where multiple entities can delegate to the same operator.

Reply:
Outdated.